pyproject.toml

```diff
@@ -76,6 +76,7 @@ concurrency = ["thread", "multiprocessing"]
 omit = [
     # don't include tests in coverage
     # "tests/*",
+    "cpp_linter/__init__.py"
```

Comment:
Its weird that this worked!

Since we're omitting the entire file, we can probably remove https://github.com/cpp-linter/cpp-linter/blob/f01a6b1fa7d049b1df3b8642b3d851be2ba46fd4/pyproject.toml#L96-L97 because that only affected `cpp_linter/__init__.py`.

I don't see this used anywhere else in the source. On the other hand, it might be handy to use for quick tests in developing new features.